app/templates/questionnaire.html

```diff
@@ -28,8 +28,8 @@
 
     {% endfor %}
 
-  <button class="btn btn--neutral" type="button" name="button">Save &amp; Exit</button>
-  <button class="btn" type="button" name="button">Save &amp; Continue</button>
+  <button class="btn btn--neutral" type="submit" name="action[save_exit]">Save &amp; Exit</button>
```

Comment:
it doesn't do anything at the moment, it was just added there so that we could differentiate between buttons clicked.